src/credit_default_eda.ipynb

```diff
@@ -66,6 +89,22 @@
    "cell_type": "markdown",
    "id": "fbc8bb3c-89ca-4b49-9391-18a2762075ce",
    "metadata": {},
+   "source": [
+    "This project aims to build a classification model using a machine learning algorithm to predict potential credit default accounts of Taiwan's credit card clients'. The data set is a Taiwan credit card data from April to September, 2005 sourced from the UCI machine learning repository and can be found [here](https://archive-beta.ics.uci.edu/ml/datasets/default+of+credit+card+clients). Although its original creators are unknown, previous studies strongly suggest it is a private data. The data set consists of 23 features column and one target column. \n",
+    "\n",
+    "There are 30,000 observations in the data set and 23 features. There are no observations with missing values in the data set. Below we show the number of each observations for each of the classes in the data set.\n",
+    "\n",
+    "|Default Payment = 0|Default Payment = 1|\n",
+    "|------|------|\n",
+    "|23364|6636|\n",
```

Comment:
do not we need a script to show how we derived to these numbers and show it as a dataframe ?

Reply:
no we don't need a script, I asked this question on 522 slack channel and Florencia said we only need the script for downloading the file. Also in Tiffany's example for milestone 1, we don't need a script.

Reply:
Actually there is a script in her Rmd file, but if you checked it with Florencia then we are good to go for summary.

Reply:
I will add some more sentences here though, like we are going to be interested in predicting if default will occur or no, because for the bank it is essential issue. They want to know what to expect and whom to trust, thus identifying customers who are not going to pay default is crucial for their record: helps bank to forecast the future steps connected not only with the individual client, but also with bank strategy as a whole. 

Something like this. @thayeylolu

Reply:
Mentioning again on this thread if we can change **Default Payment** to **Default Payment next month** as predictions are for next month. 

I also think we should have the script of how we derived these numbers even if we don't need to upload it in milestone-1 Good to keep it for our record.

Reply:
I included a cell in the eda file on how we got the numbers.

Reply:
Yes sorry I misunderstood, we will be keeping the code in the notebook